src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
```

Comment:
For better readability, it is better to put spaces for parameters. Also naming parameters with the same letter is a bad experience. Especially still put numbers in the title.
Ex: `( women ) => {}`